### ISSUE TITLE
Accounts, Passengers, Orders - Revise System Relationship

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#1843]
Most of our problems are related to price changes when edits we've done had nothing to do with price.
Can we instead segment our system as follows:

Can we have an:
Accounts (working just fine)
Passengers (sub-set of accounts)
Orders (this would have any price sensitive info)

Accounts kept as is; including name, shipping address, contact info, credit card billing address.

Passengers would include ALL custom information that does not affect price in any way; ie name as per passport, dob, passport info, dietary restrictions, ffn, known travelers, wheelchair requirements (no price attached), special email address. It would be modifiable unconnected to order – it could be modifiable even when there was no order at all. ANY passengers listed on an account would appear in the drop-down for any order that the account initiated. Information updated here would be modified for passengers on ALL upcoming trips.

Orders level would have any order specific info associated with additional cost; departure/return city, business class, pre or post tour options, deviations, complimentary bags, special mailing information.
Here is a photo of where the info would be modifiable for the client:

 Can we have an:
Accounts
Passengers (sub-set of accounts)
Orders

Accounts kept as is; including name, shipping address, contact info, credit card billing address.

Passengers would include ALL custom information that does not affect price in any way; ie name as per passport, dob, passport info, dietary restrictions, ffn, known travelers, wheelchair requirements (no price attached), special email address. It would be modifiable unconnected to order – it could be modifiable even when there was no order at all. ANY passengers listed on an account would appear in the drop-down for any order that the account initiated. Information updated here would be modified for passengers on ALL upcoming trips.

Orders level would have any order specific info associated with additional cost; departure/return city, business class, pre or post tour options, deviations, complimentary bags, special mailing information.
Here is a photo of where the info would be modifiable for the client:
Can we have an:
Accounts
Passengers (sub-set of accounts)
Orders

Accounts kept as is; including name, shipping address, contact info, credit card billing address.

Passengers would include ALL custom information that does not affect price in any way; ie name as per passport, dob, passport info, dietary restrictions, ffn, known travelers, wheelchair requirements (no price attached), special email address. It would be modifiable unconnected to order – it could be modifiable even when there was no order at all. ANY passengers listed on an account would appear in the drop-down for any order that the account initiated. Information updated here would be modified for passengers on ALL upcoming trips.

Orders level would have any order specific info associated with additional cost; departure/return city, business class, pre or post tour options, deviations, complimentary bags, special mailing information.
Here is a photo of where the info would be modifiable for the client:

 
![image](https://cloud.githubusercontent.com/assets/22382144/22486395/5a941400-e7d7-11e6-8768-cd55ee423438.png)

**Comment by bradatkins** [comment]
Something like this:

![image](https://cloud.githubusercontent.com/assets/8440255/22535730/f8624e06-e8ca-11e6-9a7b-6a6508ae8393.png)

**Comment by chris206** [comment]
Yes- something like this is the right way forward **BUT**, crucially
I want the passenger information that does not affect price to be separated from the specific order info that does affect price.

So - at the account level - all of this info is good
but the passenger section should not include any price related information - such as departure city, business class etc. I have made a very rough mock up


![image](https://cloud.githubusercontent.com/assets/22382144/22554231/5ec9a114-e92d-11e6-8319-21bf46cdaa1c.png)

In this way, the account level could have 10 passengers - each of which would appear in the drop-down - but the order level would only associate the passengers who were traveling on a given order.

So, the order level would show only the name of the passenger, and possibly the dates of birth as unmodifiable foreign keys. These would be associated to the specific order requests
